sdk/identity/Azure.Identity/src/MsalPublicClient.cs

```diff
@@ -24,10 +24,9 @@ public MsalPublicClient(HttpPipeline pipeline, string clientId, string tenantId
         {
             PublicClientApplicationBuilder pubAppBuilder = PublicClientApplicationBuilder.Create(clientId).WithHttpClientFactory(new HttpPipelineClientFactory(pipeline));
 
-            if (!string.IsNullOrEmpty(tenantId))
-            {
-                pubAppBuilder = pubAppBuilder.WithTenantId(tenantId);
-            }
+            tenantId ??= "organizations";
```

Comment:
Did you intent to use Constants.OrganizationsTenantId  here?

Reply:
yes thanks